tests/js/common/test-data/search/docs/ii_sa_dataset.json

```diff
@@ -0,0 +1 @@
en": ["51.21187", "4.25633"], "Titel": "Beveren", "istGut": true}, {"Koordinaten": ["51.40944", "46.04833"], "Titel": "Privolzhskiy"}]}, "Gro\u00dfvater": {"Name": "Sharon", "Familienname": "Lopez", "Geburtsdatum": "2014-11-11", "Setzt": []}, "Schwester": {"Name": "Lori", "Familienname": "Navarro", "Geburtsdatum": "2018-11-24", "Setzt": [{"Koordinaten": ["53.53041", "43.67663"], "Titel": "Nizhniy Lomov", "istGut": false}, {"Koordinaten": ["50.23019", "8.77155"], "Titel": "Karben", "Bezeichnung": "Necessary production page certainly detail."}, {"Koordinaten": ["44.99012", "-123.02621"], "Titel": "Keizer"}, {"Koordinaten": ["27.71809", "-82.35176"], "Titel": "Sun City Center"}, {"Koordinaten": ["50.88506", "4.07601"], "Titel": "Denderleeuw", "Bezeichnung": "Where turn good book only.", "Merkmale": ["somebody", "foreign", "two", "always", "degree", "form", "conference", "economy", "trouble", "cold"]}, {"Koordinaten": ["47.30997", "8.52462"], "Titel": "Adliswil", "Bezeichnung": "Teacher best far senior.", "istGut": false}]}, "Enkelin": {"Name": "Nina", "Familienname": "Lee", "Geburtsdatum": "2006-09-15", "Setzt": [{"Koordinaten": ["13.8196", "100.04427"], "Titel": "Nakhon Pathom", "istGut": false}, {"Koordinaten": ["14.62578", "121.12251"], "Titel": "Antipolo", "Merkmale": ["big", "toward", "bed", "woman", "note", "ready", "writer", "fast", "ready", "forward"]}, {"Koordinaten": ["22.49396", "-105.36369"], "Titel": "Acaponeta", "Merkmale": ["sign", "care", "report", "away", "situation", "mission", "image", "issue", "finish", "soon"], "Bezeichnung": "Become support impact first top paper."}, {"Koordinaten": ["51.40148", "-1.32471"], "Titel": "Newbury", "Bezeichnung": "Western car rule ground story."}, {"Koordinaten": ["48.98994", "2.1699"], "Titel": "Herblay", "Merkmale": ["easy", "behind", "maybe", "hand", "beautiful", "change", "night", "perhaps", "wide", "bed"]}, {"Koordinaten": ["23.73333", "114.68333"], "Titel": "Heyuan", "Merkmale": ["two", "know", "go", "task", "its", "can", "affect", "well", "child", "unit"]}, {"Koordinaten": ["41.15612", "1.10687"], "Titel": "Reus", "Merkmale": ["teach", "book", "great", "answer", "show", "attack", "wait", "around", "exactly", "law"]}, {"Koordinaten": ["51.955", "5.22778"], "Titel": "Culemborg"}]}, "Bruder": {"Name": "Carolyn", "Familienname": "Anderson", "Geburtsdatum": "1999-09-10", "Setzt": [{"Koordinaten": ["26.76628", "83.36889"], "Titel": "Gorakhpur", "istGut": true}, {"Koordinaten": ["47.46657", "84.87144"], "Titel": "Zaysan", "istGut": false, "Bezeichnung": "Able capital example common huge all."}, {"Koordinaten": ["50.86079", "17.4674"], "Titel": "Brzeg", "Bezeichnung": "Bag brother chance partner."}, {"Koordinaten": ["53.49389", "-1.29243"], "Titel": "Mexborough", "Merkmale": ["goal", "nation", "would", "follow", "huge", "national", "daughter", "all", "often", "enough"]}, {"Koordinaten": ["32.55531", "76.12647"], "Titel": "Chamba"}]}}, "Setzt": [{"Koordinaten": ["29.53885", "-95.44744"], "Titel": "Fr
```

Comment:
eof on new line?

Reply:
I don't think that this is required. I works now